docs/UserGuide.md

```diff
@@ -63,8 +63,23 @@ than current GUI apps in the industry.
 
 ## **Glossary**
 
+### Definitions
+
 Here are some descriptions of the words we use throughout the User Guide:
-- Parameter: Parameters are like fields in a form you are required to fill up. They include details you would include about the doctor/patient.
+- **Parameter**: Parameters are like fields in a form you are required to fill up.
+They include details you would include about the doctor/patient.
+- **Command**: An input from the user that tells Docedex to perform an action (ie. add a doctor).
+- **GUI**: Graphical User Interface (GUI) represents the visual display of Docedex that users can see.
+- **GUI component**: A subsection of the Graphical User Interface.
+- **CLI**: Command Line Interface (CLI) represents a text-based user interface to interact with the application.
+- **Doctors List**: A scrollable GUI component that displays a list of doctors.
+  Each doctor in the doctors list is encapsulated within a doctor card.
+- **Doctor Card**: A small rectangular GUI component that displays key information about
+  a doctor such as name, phone number, email and tags.
+- **Patients List**: A scrollable GUI component that displays a list of patients.
+  Each patient in the patients list is encapsulated within a patient card.
+- **Patient Card**: A small rectangular GUI component that displays key information about
+  a patient such as name, phone number, email and tags.
```

Comment:
Actually now that I think about it maybe we should change the phrasing to more user friendly names like doctor panel or information panel instead of calling it enlarged contact card

Reply:
Oh wait you are right; let me remove the redundant ones